src/libsyncengine/update_detection/update_detector/updatetreeworker.cpp

```diff
@@ -227,21 +226,20 @@ ExitCode UpdateTreeWorker::step3DeleteDirectory() {
                 }
 
                 if (!parentNode->insertChildren(newNode)) {
-                    LOGW_SYNCPAL_WARN(_logger, L"Error in Node::insertChildren: node name="
-                                                       << SyncName2WStr(newNode->name()).c_str() << L" parent node name="
-                                                       << SyncName2WStr(parentNode->name()).c_str());
+                    LOGW_SYNCPAL_WARN(_logger, L"Error in Node::insertChildren: Node name="
+                                                       << Utility::formatSyncName(newNode->name()) << L" parent Node name="
+                                                       << Utility::formatSyncName(parentNode->name()));
                     return ExitCode::DataError;
                 }
 
                 _updateTree->nodes()[deleteOp->nodeId()] = newNode;
                 if (ParametersCache::isExtendedLogEnabled()) {
-                    LOGW_SYNCPAL_DEBUG(
-                            _logger,
-                            _side << L" update tree: Node '" << SyncName2WStr(newNode->name()).c_str() << L"' (node ID: '"
-                                  << Utility::s2ws(newNode->id().has_value() ? *newNode->id() : NodeId()).c_str()
-                                  << L"', DB ID: '" << (newNode->idb().has_value() ? *newNode->idb() : -1) << L"', parent ID: '"
-                                  << Utility::s2ws(parentNode->id().has_value() ? *parentNode->id() : NodeId()).c_str()
-                                  << L"') inserted. Operation DELETE inserted in change events.");
+                    LOGW_SYNCPAL_DEBUG(_logger, _side << L" update tree: Node name="
+                                                      << Utility::formatSyncName(newNode->name()).c_str() << L" (NodeId='"
+                                                      << Utility::s2ws(newNode->id() ? *newNode->id() : NodeId()) << L"', DbId= '"
+                                                      << (newNode->idb() ? *newNode->idb() : -1) << L"', parent NodeId='"
+                                                      << Utility::s2ws(parentNode->id() ? *parentNode->id() : NodeId())
+                                                      << L"') inserted. Operation DELETE inserted in change events.");
```

Comment:
In order to uniformize even further, can we call `makeDeleteLogWMsg`with `parentNode->id()`as an optional parameter

Reply:
Or log parent ID everywhere

Reply:
@luc-guyot-infomaniak

Reply:
I'll close this PR and make another one once we have merged `release-3.6.7` into `develop` because:
- it is already difficult to merge because of current conflicts
- it is going to make the the merge of `release-3.6.7` into `develop` difficult, as numerous files are changed in numerous locations.